packages/bot-web-ui/src/components/dashboard/tutorial-tab/quick-strategy-content/quick-strategy-cards.tsx

```diff
@@ -0,0 +1,103 @@
+    ({ quick_strategy_content, tutorial_selected_strategy, setTutorialSelectedStrategy }: TQuickStrategyGuides) => {
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+
+        return (
+            <>
+                {tutorial_selected_strategy === '' ? (
+                    <div className='tutorials-quick-strategy__cards'>
+                        {quick_strategy_content?.map(({ qs_name, content, type }) => (
+                            <div
+                                className='tutorials-quick-strategy__placeholder'
+                                key={type}
+                                onClick={() => setTutorialSelectedStrategy(qs_name)}
+                                role='button'
+                                tabIndex={0}
+                            >
+                                <div>
+                                    <div className='tutorials-quick-strategy__placeholder__title'>
+                                        <Text
+                                            align='center'
+                                            weight='bold'
+                                            color='prominent'
+                                            line_height='s'
+                                            size={is_mobile ? 'xxs' : 's'}
+                                        >
+                                            {type}
+                                        </Text>
+                                    </div>
+                                    <div className='tutorials-quick-strategy__placeholder__content'>
+                                        <ul>
+                                            {content.map(data => (
+                                                <li key={data}>
+                                                    <Text
+                                                        align='center'
+                                                        color='prominent'
+                                                        line_height='s'
+                                                        size={is_mobile ? 'xxs' : 's'}
+                                                    >
+                                                        {data}
+                                                    </Text>
+                                                </li>
+                                            ))}
+                                        </ul>
+                                    </div>
+                                </div>
+                                <Icon icon='IcChevronRightBold' />
+                            </div>
+                        ))}
+                    </div>
+                ) : (
+                    <>
+                        <div className='tutorials-quick-strategy__breadcrumb'>
+                            <Text
+                                className='tutorials-quick-strategy__breadcrumb--click'
```

Comment:
I think the default class shouldn't be prefixed with interactivity `--click` by BEM
maybe it could be `className='tutorials-quick-strategy__selectable-breadcrumb'`

Reply:
Updated.